cryptolib/testdata/publicKey.gpg

```diff
@@ -0,0 +1,41 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
nit: Make these constants in the test file instead of seperate test data

Reply:
Done.